swaylock/main.c

```diff
@@ -115,14 +115,100 @@ void notify_key(enum wl_keyboard_key_state state, xkb_keysym_t sym, uint32_t cod
 	}
 }
 
+static void cairo_set_source_u32(cairo_t *cairo, uint32_t color) {
```

Comment:
Hold on, I'm not understanding why moving this one function has anything to do with gdk-pixbuf.